src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java

```diff
@@ -80,7 +81,11 @@ static String findRuntime(StageFlexibleConfiguration config)
       throws IOException, AppEngineException {
     try {
       // verification for app.yaml that contains runtime:java
-      Path appYaml = config.getAppEngineDirectory().toPath().resolve(APP_YAML);
+      File appEngineDirectory = config.getAppEngineDirectory();
+      if (appEngineDirectory == null) {
+        throw new AppEngineException("Malformed app.yaml: missing App Engine directory");
```

Comment:
I think this should be "Invalid Staging Configuration: missing App Engine directory" or something, it doesn't have anything to with the contents of app.yaml?

Reply:
done